### PR TITLE
Fix transiently failing tool shed tests.

### PR DESCRIPTION
**Pull request by jmchilton**
Comments inline - I don't know why but the retry would fix the transient problems acording to the Jenkins tests with the previous exception added to verify this would fix the problem.

### CODE REVIEW

**Comment by martenson** [comment]
black magic I say!

**Comment by dannon** [comment]
If there's a valid failure here in the naked exception, won't this go into an infinitely failing loop?  (that is, maybe `aggressive=False` on the second attempt)

**Comment by jmchilton** [comment]
@dannon `export_capsule` calls `_export_capsule` not `export_capsule`, I don't think it will result in an infinite loop - let me know if I am wrong though.

File: test/tool_shed/base/twilltestcase.py [hunk 1]
```diff
@@ -548,18 +548,11 @@ def export_capsule( self, repository, aggressive=False, includes_dependencies=No
         else:
             try:
                 return self._export_capsule(repository, includes_dependencies=includes_dependencies)
-            except Exception as original_error:
+            except Exception:
+                # Emperically this fails occasionally, we don't know
```
Comment (dannon) [inline test/tool_shed/base/twilltestcase.py]: 'Empirically'

Comment (jmchilton) [inline test/tool_shed/base/twilltestcase.py]: Fixed.

**Comment by dannon** [comment]
Argh, missed that `_`.  Of _course_ it makes sense to have export_capsule _and_ _export_capsule.  This should be fine.

**Pull request merged**